src/shitalSDec19/assignment2_resubmission/Solution1.java

```diff
@@ -0,0 +1,32 @@
+/* 1) Find the missing number in integer array of 1 to 10? 
+ 
+Hint: You have given an integer array which contains numbers from 1 to 10 but one number is missing, you need to write a Java program to find that missing number in an array.  */
+
+package shitalSDec19.assignment2_resubmission;
+
+public class Solution1 {
+	int sum = 0;
+	int missingNo = 0;
+	int maxNo = 0;
+	int arrSum = 0;
+
+	void findMissingNo(int[] arr) {
+		for (int i = 0; i < arr.length; i++) {
+			if (maxNo < arr[i]) {
```

Comment:
Why we were checking every time maxNo?
this is not efficient coding @ShitalSDec19